enhancements/sig-architecture/26-policy-generator/README.md

```diff
@@ -0,0 +1,253 @@
+        patches:
+            # Optional: Only required when there are multiple manifests in the path.
+          - apiVersion: ""
+            # Optional: Only required when there are multiple manifests in the path.
+            kind: ""
+            metadata:
+              # Optional: Only required when there are multiple manifests in the path.
+              name: ""
+              # Optional: Only required when there are multiple manifests in the path and it's a
+              # manifest to a namespaced resource.
+              namespace: ""
+              # An example modification to the manifest
+              annotations:
+                friends-character: Chandler Bing
+    # Optional. (See policyDefaults.categories for description.)
+    categories:
+      - "CM Configuration Management"
+    # Optional. Determines the policy controller behavior when comparing the manifest to objects on
+    # the cluster ("musthave",  "mustonlyhave", or "mustnothave"). Defaults to "musthave".
+    complianceType: "musthave"
+    # Optional. (See policyDefaults.controls for description.)
+    controls:
+      - "CM-2 Baseline Configuration"
+    # Optional. (See policyDefaults.disabled for description.)
+    disabled: false
+    # Optional. (See policyDefaults.informKyvernoPolicies for description.)
+    informKyvernoPolicies: true
+    # Optional. (See policyDefaults.consolidateManifests for description.)
+    consolidateManifests: true
+    # Optional. Determines the list of namespaces to check on the cluster for the given manifest. If
+    # a namespace is specified in the manifest, the selector is not necessary. This defaults to no
+    # selectors.
+    namespaceSelector:
+      include: []
+      exclude: []
+    # Optional. (See policyDefaults.placement for description.)
+    placement: {}
+    # Optional. (See policyDefaults.remediationAction for description.)
+    remediationAction: ""
+    # Optional. (See policyDefaults.severity for description.)
+    severity: "low"
+    # Optional. (See policyDefaults.standards for description.)
+    standards:
+      - "NIST SP 800-53"
+```
+
+
+#### Example Workflow
+
+- A security engineer, SRE, infrastructure engineer, or etc. creates a pull request (PR) with the
+  Kubernetes resource object manifests that should be wrapped in a policy.
+- Based on the organization policy, the PR is reviewed and merged in the organization's Git
+  repository.
+- A security engineer, SRE, infrastructure engineer, or etc. creates a PR with the
+  `kustomization.yaml` file, the generator plugin configuration, and any Kustomize patches they
+  would like to apply to the generated policies.
+- Based on the organization policy, the PR is reviewed and merged in the organization's Git
+  repository.
+- A SRE then configures an ACM subscription to apply the generated policies using GitOps. This
+  subscription would be limited to just the policy directory created in the previous step.
```

Comment:
I would like to see more examples on how it can be used. For instance some configuration examples under different scenarios.